src/adapters/communities-db.ts

```diff
@@ -600,7 +601,7 @@ export function createCommunitiesDBComponent(
 
     async updateCommunity(
       communityId: string,
-      updates: Partial<Pick<CommunityDB, 'name' | 'description' | 'private'>>
+      updates: Partial<Pick<CommunityDB, 'name' | 'description' | 'private' | 'needs_manual_review' | 'active'>>
```

Comment:
We shouldn't allow updating `needs_manual_review` for now, right?
```suggestion
      updates: Partial<Pick<CommunityDB, 'name' | 'description' | 'private' | 'active'>>
```